app/models/card.rb

```diff
@@ -10,12 +10,25 @@ class Card < ActiveRecord::Base
   mount_uploader :image, ImageUploader
 
   def set_default_review_date
-    self.review_date = self.review_date + 3.day
+    self.review_date = DateTime.now
   end
 
   def check_translation(input_text)
-    if prepare_text(translated_text) == prepare_text(input_text)
-      self.update_attributes(review_date: (DateTime.current+3.day).to_date)
+    if prepare_text(translated_text) ==
+        prepare_text(input_text) && self.accuracy <= -3
```

Comment:
Align the operands of a condition in an `if` statement spanning multiple lines.<br>Redundant `self` detected.